app/views/relationships/index.html.haml

```diff
@@ -0,0 +1,22 @@
+%section.people
+  %header
+    %h2 People I Follow
+  %table.table
+    %thead
+      %tr
+        %th(width="30%")
+        %th(width="20%") Videos in Queue
+        %th(width="20%") Followers
+        %th(width="30%") Unfollow
+    %tbody
+      - @leaders.each do |leader|
+        %tr
+          %td
+            %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest(leader.email.downcase)}?s=40")
+            = link_to leader.full_name, leader
+          %td.extra-padding= leader.queue_items.count
+          %td.extra-padding= leader.followers.count
+          %td.extra-padding
+            - relationship = Relationship.find_by(leader: leader, follower: current_user)
```

Comment:
I don't recommend using business logic like this in your views, though I realize that this is only one line. I recommend trying something a bit more compact. For instance, you could do something like  
`= link_to Relationship.where(leader: leader, follower: current_user)`
or using find_by directly as that first argument.
You could also make a helper method in under relationships helpers folder.